src/main/java/homework6/Main.java

```diff
@@ -0,0 +1,123 @@
+package src.main.java.homework6;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+        compareNumbersBool(10, 12);
+        compareNumbersBool(1, 2);
+        compareNumbersBool(10, 0);
+        compareNumbersBool(10, 5);
+        compareNumbersBool(10, 10);
+        isPositiveNumber(-2);
+        isPositiveNumber(0);
+        isPositiveNumber(32);
+        printString("Homework is done!", 5);
+        isPositiveNumberBool(-1);
+        isPositiveNumberBool(0);
+        isPositiveNumberBool(3);
+        leapYearBool(2028);
+        leapYearBool(4);
+        leapYearBool(100);
+        leapYearBool(400);
+        leapYearBool(600);
+
+
+    }
+
+    static void printThreeWords() {
+        System.out.println("\nOrange.\nBanana.\nApple.");
+    }
+
+    static void checkSumSign() {
+        int a = -150;
+        int b = 100;
+
+        if (a + b >= 0) {
+            System.out.println("\nSum is positive: " + (a + b));
+        } else {
+            System.out.println("\nSum is negative: " + (a + b));
+        }
+    }
+
+
+    static void printColor() {
+        int a = 101001001;
+
+        if (a <= 0) {
+            System.out.println("\nЧервоний");
+        } else if (a > 0 && a <= 100) {
+            System.out.println("\nЖовтий");
+        } else {
+            System.out.println("\nЗелений");
+        }
+    }
+
+    static void compareNumbers() {
+        int a = 32;
+        int b = 21;
+        if (a >= b) {
+            System.out.println("\na >= b");
```

Comment:
System.out.println() - adds new line symbol - there is no need is addition `\n`

Reply:
done in purose of better visual while reading output in console